explain.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, explain_map):
             "Hash Join": self._explain_hj,
             "Seq Scan": self._explain_ss,
             "Hash": self._explain_hash,
+            "Sorted Merge": self._explain_sorted_merge
```

Comment:
There is no such node type. Sorted merge is implemented as 2 diff steps. Refer to this plan: https://explain.dalibo.com/plan/75a34892f7247f55. Check out the raw node type value, the string should match that. (case sensitive).